packages/schemas/tables/users.sql

```diff
@@ -11,6 +11,7 @@ create table users (
   avatar varchar(2048),
   application_id varchar(21),
   role_names jsonb /* @use RoleNames */ not null default '[]'::jsonb,
+  account_status boolean not null default true,
```

Comment:
Should we give it a more precise field name like `is_suspended` or `is_deleted`? Since it's a boolean value, `account_status` more like an enum value?